### ISSUE TITLE
skool2html: Error while parsing #R macro

### ISSUE THREAD

**Issue by chupocro** [skoolkid/skoolkit#108]
I am following the [Getting started](https://skoolkit.ca/docs/skoolkit/diy.html) tutorial and everything up to (and including) **creating HTML disassembly from skool file** works OK but in the step where it is shown how to generate the HTML disassembly using the .ref file:

`python skool2html.py examples\hungry_horace.ref`

there is an error saying:

> Error while parsing #R macro: Could not find instruction at 16384

Is this maybe an old tutorial that is not working with the most recent Skoolkit?

**Comment by skoolkid** [comment]
Yes, the tutorial is a bit out of date. Oops! The command you need to run is:

    python skool2html.py hungry_horace.skool examples\hungry_horace.ref

I'll fix this for the next version of SkoolKit.

**Comment by chupocro** [comment]
That was fast  :-) Yes, now it works well, thank you very much! I was disassembling Spectrum's games back in the 80s using just tape recorder, MONS (GENS and Zeus assembler), paper and pencil (I spent months disassembling the music routine from Imagine/Konami's Ping Pong and I still have a bunch of papers with hand commented code somewhere). I was looking for some Spectrum game disassemblies and found SkoolKit so now I might try to disassemble some interesting routines from some of the games (e.g. the handwriting intro from Video Pool or similar) to see how it will be going. I am not sure if emulator's debuggers will be suitable to track the entry points, back in the days I had to follow what's happening after loading to determine the address where the m/c starts but now it could be easier to use breakpoints and executing the program step by step.

I am surprised not many games are already disassembled by others. Seems as there are more arcade machine games disassembled than Spectrum games which is surprising because almost every arcade machine has different hardware which has to be examined before analyzing the ROMs.

Thank you one more time,
Best regards
